src/components/autocomplete/autocomplete.spec.js

```diff
@@ -98,6 +98,31 @@ describe('<md-autocomplete>', function() {
       element.remove();
     }));
 
+    it('should allow receiving focus on the autocomplete', function() {
+      var scope = createScope(null, {inputId: 'custom-input-id'});
+      var template = '\
```

Comment:
I'm just trying my best in using the best code conventions. The tests above also use `\`.
But I will use string concatenation, because you wish that :smile: